pkg/parser/terraform/data_source.go

```diff
@@ -290,3 +292,11 @@ func parseDataSourceBody(body *hclsyntax.Body) string {
 	}
 	return buffer.String()
 }
+
+func deleteResourcesFromPolicy(body *hclsyntax.Body) {
```

Comment:
I think that is not a "good idea" to delete `resources` from policy because of future queries that will need to look at this field.

KICS already have some queries that look to `Resource`. It something that we should look carefully. As examples:
- kics\assets\queries\terraform\aws\iam_role_with_full_privileges
- kics\assets\queries\terraform\aws\iam_role_policy_passrole_allows_all
- kics\assets\queries\terraform\aws\iam_policy_grants_full_permissions